packages/mobile/src/mock/mock-images.ts

```diff
@@ -0,0 +1,5 @@
+const MOCK_IMAGES = {
+  JYSK: require('~/mock/jysk.png'),
```

Comment:
I don't think we need this. Let's just show the circle with seller's first letter in case there is no image provided.
You may modify `Avatar` component for that.

Reply:
fixed